libs/s25main/nodeObjs/noStaticObject.h

```diff
@@ -37,8 +37,10 @@ class noStaticObject : public noCoordBase
     void Draw(DrawPoint drawPt) override;
 
     static Textures getTextures(unsigned short file, unsigned short id);
+    bool IsAnimated() const;
```

Comment:
One small inconsistency: `IsAnimated` vs `isOpenGateway` - I think the `I` should be lower case. but this is very picky, I know :-)

That the only thing left I think. Thank you :-)

Reply:
Changed to upper case because all other functions are named this way. (IsOpenGateway)